app/src/styles/tringo.scss

```diff
@@ -1,52 +1,75 @@
 @import 'tringo.mobile';
 
 .App {
-    text-align: center;
-    min-height: 768px;
+  text-align: center;
```

Comment:
what's wrong with your IDE again?!
I won't merge it unless there're 2-spaces tabs present. fix it asap

Reply:
css for Webstorm settings are off